### DISCUSSION TITLE
Danke und Versionsinfo

### DISCUSSION THREAD

**Discussion by jos-webservices** [xgcssch/SystaComfortPrometheusExporter#15] [General]
Hallo Sönke,
ein ganz großes Dankeschön!!!!! Endlich kann ich die Daten meiner SystaComfort mitschneiden, darauf hab ich jetzt schon drei Jahre gewartet! Ich habe auch alles auf einem Synology NAS eingerichtet, seit heute wird aufgezeichnet.
Ich wollte noch kurz was zur Kompatibilitätsübersicht beitragen, bei mir läuft das Aufzeichnen problemlos mit dieser Firmware in der Anlage; 
![Screenshot_20211112-223513_S-Touch](https://user-images.githubusercontent.com/62727059/141537621-604c8fa0-d67e-4b56-970d-b816a5462787.jpg)

Als nächsten Schritt will ich jetzt noch die Daten vom Kessel importieren und die meiner Wetterstation aufzeichnen... Mal sehen wie weit ich komme.

Viele Grüße
Jochen

**Comment by xgcssch** [comment]
Danke Jochen!
Habe ich in die Readme eingearbeitet.
Dort habe ich auch einen Hinweis eingefügt, dass die Möglichkeit besteht, Anlagen mit Hardwarestand 2 über Modbus TCP zu steuern.
Gruß
Sönke
